static/app/components/replays/replayContext.tsx

```diff
@@ -206,6 +206,7 @@ export function Provider({children, replay, initialTimeOffset = 0, value = {}}:
   const [fastForwardSpeed, setFFSpeed] = useState(0);
   const [buffer, setBufferTime] = useState({target: -1, previous: -1});
   const playTimer = useRef<number | undefined>(undefined);
+  const unMountedRef = useRef<boolean>(false);
```

Comment:
Don't need to be specific about the type, TS can infer it from use in this case

```suggestion
  const playTimer = useRef<number | undefined>(undefined);
  const unMountedRef = useRef(false);
```

In the `playTimer` case the type isn't inferred because it's started out with `undefined` but getting the number later on.